language/stdlib/modules/ValidatorConfig.move

```diff
@@ -13,13 +13,8 @@ module ValidatorConfig {
 
     struct Config {
         consensus_pubkey: vector<u8>,
-        /// TODO(philiphayes): restructure
-        ///   3) remove validator_network_identity_pubkey
-        ///   4) remove full_node_network_identity_pubkey
-        validator_network_identity_pubkey: vector<u8>,
-        validator_network_address: vector<u8>,
-        full_node_network_identity_pubkey: vector<u8>,
-        full_node_network_address: vector<u8>,
+        validator_network_addresses: vector<u8>,
```

Comment:
Would you mind adding some Move tests that set `validator_network_addresses` and `fullnode_network_addresses` to non-empty garbage that won't parse as a list of (encrypted or non-encrypted) network addresses? Thinking about that nasty crash we had [here](https://github.com/libra/libra/blob/master/execution/executor/src/lib.rs#L353) and want to make sure we cover that case...

Reply:
Spoke offline, as this is just a raw move value, the test is redundant. @zekun000 had a proposal to make this richer -- if we do -- such tests are critical.

At the same time, I think we should investigate a thoroughness check across all move resource / on chain configs in Rust and ensure that nothing nasty happens upon deserialization and / or that the input and output are symmetric.